common/djangoapps/util/tests/test_request.py

```diff
@@ -50,8 +50,25 @@ def test_course_id_from_url(self):
         self.assertIsNone(course_id_from_url('/login'))
         self.assertIsNone(course_id_from_url('/course/edX/maths/2020'))
         self.assertIsNone(course_id_from_url('/courses/edX/maths/'))
+        self.assertIsNone(course_id_from_url('/api/courses/v1/blocks/edX/maths/2020'))
```

Comment:
Nit: Add new course-id format in addition to the old (Mongo) course-id format for the `blocks` url.